src/uwtools/utils/tasks.py

```diff
@@ -229,8 +229,12 @@ def hardlink(
     try:
         os.link(src, dst)
     except Exception as e:
-        if symlink_fallback:
+        if fallback == STR.symlink:
             os.symlink(src, dst)
+            log.info("Could not hardlink %s -> %s, symlink instead" % (dst, src))
```

Comment:
```suggestion
            log.info("Could not hardlink %s -> %s, symlinked instead" % (dst, src))
```
Since it has already happened by the time this is logged...